micro-rdk/src/common/config_monitor.rs

```diff
@@ -1,19 +1,39 @@
 use super::app_client::{AppClient, AppClientError, PeriodicAppClientTask};
+#[cfg(feature = "provisioning")]
```

Comment:
I think you should remove this now, since the types are no longer gated on provisioning.